vmcheck/vmcheck.go

```diff
@@ -24,21 +24,26 @@ import (
 // Get the CPU ID low level leaf values.
 func cpuid_low(arg1, arg2 uint32) (eax, ebx, ecx, edx uint32)
 
-// IsVirtualWorld returns true if running in a VM and the backdoor is available.
-func IsVirtualWorld() (bool, error) {
+// IsVirtualWorld returns `true` if running in a VM and the backdoor is available.
+// It also tries to elevate I/O privileges for the calling thread, which in
+// some cases may be forbidden by the system (e.g Linux in `kernel_lockdown` mode
+// does not allow `iopl` calls); the `ignoreAccessErrors` parameter allows
+// to control library behavior in order to threat such errors as non-fatal.
```

Comment:
```suggestion
// to control library behavior in order to treat such errors as non-fatal.
```

Reply:
Good catch! I've directly amended the typo in the existing commit.